src/Sarif/CodeGenHints.json

```diff
@@ -365,6 +365,16 @@
       "kind": "DictionaryHint"
     }
   ],
+  "run": [
```

Comment:
The JSON schema-to-.NET code generator accepts (in addition to the schema file) a "hint" file that can modify the resulting generated code. In this case, we add a hint to the `run` object which tells it that the generated `Run` class should be declared to derive from `PropertyBagHolder`. See `Run.cs` for the result.